zuul/layout.yaml

```diff
@@ -0,0 +1,87 @@
+pipelines:
+  - name: check
+    description: Newly uploaded patchsets enter this pipeline to receive an initial +/-1 Verified vote from Windows CI.
+    failure-message: Build failed.
+    manager: IndependentPipelineManager
+    precedence: low
+    trigger:
+      gerrit:
+        - event: patchset-created
+        - event: change-restored
+        - event: comment-added
+          comment_filter: (?i)recheck(( (?:bug|lp)[\s#:]*(\d+))|( no bug))(\s+clean)?\s*$
+        - event: comment-added
+          require-approval:
+            - username: jenkins
+              older-than: 72h
+        - event: comment-added
+          require-approval:
+            - approved: 1
+            - username: jenkins
+              older-than: 24h
+        - event: comment-added
+          comment_filter: (?i)^\s*reverify( (?:bug|lp)[\s#:]*(\d+))\s*$
+          require-approval:
+            - approved: 1
+            - username: jenkins
+        - event: comment-added
+          require-approval:
+            - verified: -1
+              username: jenkins
+          approval:
+            - approved: 1
+        - event: comment-added
+          require-approval:
+            - verified: -2
+              username: jenkins
+          approval:
+            - approved: 1
+    start:
+      gerrit:
+        verified: 0
+    success:
+      gerrit:
+        verified: 1
+    failure:
+      gerrit:
+        verified: -1
+
+jobs:
+  - name: ^.*$
+
+projects:
+  - name: Juniper/contrail-controller
+    check:
+      - WinCI-Proxy
+
+  - name: Juniper/contrail-vrouter
+    check:
+      - WinCI-Proxy
+
+  - name: Juniper/contrail-build
+    check:
+      - WinCI-Proxy
+
+  - name: Juniper/contrail-packages
+    check:
+      - WinCI-Proxy
+
+  - name: Juniper/contrail-third-party
+    check:
+      - WinCI-Proxy
+
+  - name: Juniper/contrail-generateDS
+    check:
+      - WinCI-Proxy
+
+  - name: Juniper/contrail-sandesh
+    check:
+      - WinCI-Proxy
+
+  - name: Juniper/contrail-docker
+    check:
+      - WinCI-Proxy
+
+  - name: Juniper/contrail-common
```

Comment:
Will this list need to contain each project on Gerrit?

Reply:
Yes.